packages/types/src/index.ts

```diff
@@ -122,6 +122,21 @@ export type ChangelogFunctions = {
   getDependencyReleaseLine: GetDependencyReleaseLine;
 };
 
+export type GetAddLine = (
+  changeset: Changeset,
+  commitOptions: null | Record<string, any>
+) => Promise<string>;
+
+export type GetVersionLine = (
+  releasePlan: ReleasePlan,
+  commitOptions: null | Record<string, any>
+) => Promise<string>;
+
+export type CommitFunctions = {
+  getAddLine: GetAddLine;
+  getVersionLine: GetVersionLine;
```

Comment:
IMHO using "message" here makes more sense as that's how this is called by git itself (https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--mltmsggt)
```suggestion
  getAddMessage: GetAddLine;
  getVersionMessage: GetVersionLine;
```

You'd have to adjust other places too (and names of those types here).